static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -125,9 +125,16 @@ function SampleImagesChartPanelBody(props: {
   return (
     <ImageWrapper>
       {images.map(resource => {
+        const hasRawDomain = Boolean(resource[RAW_DOMAIN]);
+        const isRelativeUrl = resource[SPAN_DESCRIPTION].startsWith('/');
+
+        const src =
+          isRelativeUrl && hasRawDomain
+            ? `${resource[RAW_DOMAIN]}${resource[SPAN_DESCRIPTION]}`
+            : resource[SPAN_DESCRIPTION];
         return (
           <ImageContainer
-            src={resource[SPAN_DESCRIPTION]}
+            src={new URL(src).toString()}
```

Comment:
This can throw, we should account for error cases here

Reply:
👍 good point, I also looked into the api, I think we should pass in the base url param into the URL constructor too i.e new`URL(domain, span[description]`) instead of constructing the string.

 in case there are any issues with slashes and stuff for example.
i.e 
<img width="524" alt="image" src="https://github.com/getsentry/sentry/assets/44422760/19669d70-0fae-4fae-aa4a-f16896b49897">

Reply:
@AbhiPrasad I've made the changes!